src/main/java/seedu/address/model/task/Task.java

```diff
@@ -56,9 +58,9 @@ public Task(Module module, TaskDescription description, TaskStatus status) {
         this.module = module;
         this.description = description;
         this.status = status;
-        priorityTag = null;
-        deadlineTag = null;
-        linkedExam = null;
+        this.priorityTag = null;
```

Comment:
There isn't a need for the "this" keyword since there is no variable called priorityTag here